hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -132,11 +133,21 @@ spec = around showLogsOnFailure $ do
           withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
             publishHydraScriptsAs node Faucet
               >>= timedTx tmpDir tracer node
-      it "commits from external wallet" $ \tracer -> do
+      fit "commits from external utxo" $ \tracer -> do
         withClusterTempDir "single-commits-from-external" $ \tmpDir -> do
           withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
             publishHydraScriptsAs node Faucet
               >>= singlePartyCommitsFromExternal tracer tmpDir node
+      fit "commits from external script utxo" $ \tracer -> do
+        withClusterTempDir "single-commits-script-from-external" $ \tmpDir -> do
+          withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
+            publishHydraScriptsAs node Faucet
+              >>= singlePartyCommitsFromExternalScript tracer tmpDir node
+      it "can't commit externally internal wallet utxo" $ \tracer -> do
```

Comment:
I would make this test a BehaviorSpec test so that there is less plumbing involved and keep only a very basic test here.

Reply:
See https://github.com/input-output-hk/hydra/pull/903#discussion_r1229400026